.github/workflows/check-python-dependencies.yml

```diff
@@ -0,0 +1,24 @@
+name: Check Python dependencies
+on:
+  pull_request:
+    paths:
+      - 'pyproject.toml'
+      - 'poetry.lock'
+
+jobs:
+  check-python-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```

Comment:
Our exporter images are based on the python 3.9 base ubi8 image, do we really want to have this against 3.11 or should we stick with python version that is shipped in quay ?:

```
ubi8/python-39
```

Reply:
since we have the restriction in pyproject.toml that python needs to be 3.9 or higher, I think the python version we run this job is not a problem

But I checked and we have jobs that run only on 3.9, only in 3.11, in 3.10 and 3.11 and in 3.9, 3.10 and 3.11, we should standardize this